torch/fx/graph.py

```diff
@@ -1348,6 +1348,15 @@ def node_copy(self, node: Node, arg_transform: Callable[[Node], 'Argument'] = la
         assert isinstance(kwargs, dict)
         result_node = self.create_node(node.op, node.target, args, kwargs, node.name, node.type)
         result_node.meta = copy.copy(node.meta)
+        NUMERIC_DEBUG_HANDLE_KEY = "_numeric_debug_handle"
+        if NUMERIC_DEBUG_HANDLE_KEY in node.meta:
+            numeric_debug_handle = node.meta[NUMERIC_DEBUG_HANDLE_KEY]
+            for k, v in dict(numeric_debug_handle).items():
+                if k == "output":
+                    result_node.meta[NUMERIC_DEBUG_HANDLE_KEY][k] = v
+                else:
+                    result_node.meta[NUMERIC_DEBUG_HANDLE_KEY][arg_transform(k)] = v
+
```

Comment:
@jerryzh168 I guess it might be a better idea to make this piece of logic done through a registration, so that we can allow arbitrary metadata to be preserved on a FX graph.

Reply:
We can sort out the details together if you want.

Reply:
yeah sure, we had some discussion on numeric suite today, and we're planning to change this logic to just preserve a simple id for each node for now, so it will be a bit less complicated

Reply:
> We can sort out the details together if you want.

do you need any inputs from me to come up with the design for the registration?